scene/main/http_request.cpp

```diff
@@ -572,7 +572,15 @@ void HTTPRequest::_timeout() {
 
 void HTTPRequest::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("request", "url", "custom_headers", "ssl_validate_domain", "method", "request_data"), &HTTPRequest::request, DEFVAL(PackedStringArray()), DEFVAL(true), DEFVAL(HTTPClient::METHOD_GET), DEFVAL(String()));
+	BIND_METHOD_ERR_RETURN_DOC("request", ERR_UNCONFIGURED);
+	BIND_METHOD_ERR_RETURN_DOC("request", ERR_BUSY);
+	BIND_METHOD_ERR_RETURN_DOC("request", ERR_CANT_CONNECT);
+	BIND_METHOD_ERR_RETURN_DOC("request", ERR_INVALID_PARAMETER);
```

Comment:
```suggestion
	BIND_METHOD_ERR_RETURN_DOC("request", ERR_UNCONFIGURED, ERR_BUSY, ERR_CANT_CONNECT, ERR_INVALID_PARAMETER);
```

Multiple error codes should be listed in the singe `BIND_METHOD_ERR_RETURN_DOC`, or it will pick up only the last one.